infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/AddNodeWorkflow.java

```diff
@@ -151,11 +151,17 @@ protected void stateTransfer(Set<String> endpoints, CorfuRuntime runtime,
                 ; chunkStart = chunkStart + CHUNK_SIZE) {
             long chunkEnd = Math.min((chunkStart + CHUNK_SIZE - 1), segment.getEnd() - 1);
 
+            long ts1 = System.currentTimeMillis();
+
             Map<Long, ILogData> dataMap = runtime.getAddressSpaceView()
                     .cacheFetch(ContiguousSet.create(
                             Range.closed(chunkStart, chunkEnd),
                             DiscreteDomain.longs()));
 
+            long ts2 = System.currentTimeMillis();
+
+            log.info("stateTransfer: read {}-{} in {} ms", chunkStart, chunkEnd, (ts2 - ts1));
```

Comment:
It is good for now but maybe it makes sense for the future to have a metric for this rather than having benchmarking statements frequently interleaving.